### PR TITLE
variable name changes for Issue #4141

### PR DESCRIPTION
**Pull request by NSanjay**
Hi

Let me know if additional changes are required for Issue #4141. Thank you for this awesome repository.

### CODE REVIEW

**Comment by LysandreJik** [comment]
Hi, I'm not sure we want to change this. I do agree that `adder` is more explicit than `attention_mask` and better models what it does. However, imo the API isn't limited to high-level modules but even to lower level modules such as `AlbertAttention` and `AlbertTransformer`.

These modules may be used by users in their specific applications: `from transformers.modeling_albert import AlbertTransformer`.

I think the gain here is not worth the loss of compatibility with all previous versions. What do you think @patrickvonplaten, @thomwolf, @julien-c ?

**Comment by patrickvonplaten** [comment]
Yes, I agree...sorry @NSanjay  I didn't think this fully through when answering here: https://github.com/huggingface/transformers/issues/4141#issuecomment-629875496 :-/
